samcli/commands/validate/validate.py

```diff
@@ -131,6 +132,7 @@ def _lint(ctx: Context, template: str) -> None:
 
     cfn_lint_logger = logging.getLogger("cfnlint")
     cfn_lint_logger.propagate = False
+    EventTracker.track_event("UsedFeature", "CFNLint")
```

Comment:
I think it is miss on our end for not using enums but using strings here. We may update as a follow-up PR